src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -60,11 +61,11 @@ public class ReleasesController : ControllerBase
         }
 
         [HttpDelete("release/{releaseVersionId:guid}")]
-        public async Task<ActionResult<ReleaseViewModel>> DeleteRelease(Guid releaseVersionId)
+        public async Task<ActionResult> DeleteRelease(Guid releaseVersionId)
```

Comment:
I would name this "DeleteReleaseVersion" given that it's not the full Release that you'll be deleting here.  I know that there are other areas where we simply say "Release", but I think we're trying to generally move over to renaming things correctly to ReleaseVersion where we can (this is historically because we didn't used to have separate Release and ReleaseVersion entities).